wgsl/index.bs

```diff
@@ -2155,6 +2155,20 @@ The following table lists the correspondence between WGSL texel formats and
     E.g. you can have an image with texels with 8bit unorm components, but when you sample
     them you get a 32-bit float result (or vec-of-f32).
 
+### External Sampled Texture Types ### {#external-texture-type}
+
+<pre class='def'>
+`texture_external`
+</pre>
+
+`texture_external` is an opaque type which can encapsulate multiple planes of data (e.g. RGBA, Y+UV),
+metadata about how to convert coordinates before sampling those planes (crop and rotation),
+and metadata about how to convert the sample results into RGBA or luminance data.
+
```

Comment:
The coordinates are always 2D, so it seems that should be called out in the definition.

So suggest inserting "2D" as in:

>  `texture_external` is an opaque type which can encapsulate multiple 2D planes of data data